src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,75 @@
+package edu.kis.vh.nursery;
+
+/**
+ * The type Default counting out rhymer.
+ */
+public class DefaultCountingOutRhymer {
+
+    public static final int EMPTY = -1;
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.